pr-data.csv

```diff
@@ -1676,7 +1676,7 @@ https://github.com/stanfordnlp/CoreNLP,08f6dca23e5296d6affdc40895bf294d9b3144f6,
 https://github.com/stanfordnlp/CoreNLP,08f6dca23e5296d6affdc40895bf294d9b3144f6,.,edu.stanford.nlp.semgraph.semgrex.SemgrexTest.testEqualsRelation,ID,,,
 https://github.com/stanfordnlp/CoreNLP,08f6dca23e5296d6affdc40895bf294d9b3144f6,.,edu.stanford.nlp.semgraph.semgrex.SemgrexTest.testNotEquals,ID,,,
 https://github.com/stanfordnlp/CoreNLP,08f6dca23e5296d6affdc40895bf294d9b3144f6,.,edu.stanford.nlp.util.CollectionValuedMapTest.testAddRemove,ID,,,
-https://github.com/stargate/stargate,1bb3eae8bce3eac8a54ced8054180932312b16f3,persistence-api,io.stargate.db.schema.ColumnTest.testToStringWithCollections,ID,Opened,https://github.com/stargate/stargate/pull/1383,
+https://github.com/stargate/stargate,1bb3eae8bce3eac8a54ced8054180932312b16f3,persistence-api,io.stargate.db.schema.ColumnTest.testToStringWithCollections,ID,Accepted,https://github.com/stargate/stargate/pull/1383,
```

Comment:
@arianacai1997 The PR from this row does not appear to be Accepted. I have changed it back to Opened now. You may want to follow up with the developer on accepting your fix as they didn't seem to have any problems with it.